docs/user_guide/installation.rst

```diff
@@ -68,29 +74,77 @@ From the command shell, enter the following sequence to create and activate a vi
+--------------------------
+
+.. _SC Install:
+
+
+After you've got the python dependencies installed, you will need to install SiliconCompiler. There are a few different ways to do this:
+
+1. The recommended method is to install the last stable version published to `pypi.org <https://pypi.org>`_, or
+2. You can do an offline install with a tarball (for Linux only), or
+3. You can install directly from the git repository (best for developers).
+
+Install from pypi.org 
+^^^^^^^^^^^^^^^^^^^^^
+SiliconCompiler can be installed directly from `pypi.org <https://pypi.org>`_ using pip. Activate your `Python Virtual Environment <https://docs.python.org/3/library/venv.html>`_ and follow the instructions below. (identical for Windows, Linux, and macOS).
 
 .. code-block:: bash
 
  (venv) pip install --upgrade pip                # upgrade pip in virtual env
  (venv) pip list                                 # show installed packages in venv
  (venv) pip install --upgrade siliconcompiler    # install SiliconCompiler in venv
  (venv) python -m pip show siliconcompiler       # will display  SiliconCompiler package information
- (venv) python -c "import siliconcompiler;print(siliconcompiler.__version__)"
 
-The expected version should be printed to the display:
+Skip to `confirm your installation`_.
 
-.. parsed-literal::
+Offline Install (Linux only)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+We also provide packages that bundle SC with all of its Python dependencies to enable installation on machines without an external internet connection. 
 
-   \ |release|
+To access them:
+
+1. Go our  `builds page <https://github.com/siliconcompiler/siliconcompiler/actions/workflows/wheels.yml>`_. 
+2. Click on the most recent, passing Wheels package. This should be the first green-colored build in the list.
+3. On the bottom of that page, you will see an "Artifacts" section. Click on the "artifact" to download it.
+4. The packages are named ``scdeps-<pyversion>.tar.gz``, depending on which Python version they are associated with.
+
+To install from a bundle, create a Python virtual environment following the instructions above, then perform the following commands.
+
+.. code-block:: bash
+
+   tar -xzvf scdeps-<pyversion>.tar.gz
+   pip install --upgrade pip --no-index --find-links scdeps
+   pip install siliconcompiler --no-index --find-links scdeps
+
+.. note::
+
+Note 
+This only installs the SC build system offline. Before you can start running SiliconCompiler, you will also need to make sure you have installed external PDKs and tools required to build (synthesis, place and route, etc). Typically, users of this flow have already set up their own tools and PDKs. If that is the case, you may skip ahead to  `confirm your installation`_. But if you do not have PDKs and tools set up, have an internet connection, and would like to leverage the PDKs and external tool install information from the SiliconCompiler GitHub repo, you can, by following these steps:
```

Comment:
I don't think the PDKs are in the wheel builds, so this applies to pip installs too.